### PR TITLE
remove travis from next branch

### LINKED ISSUE #1658: Remove Travis CI from next branch
As title says. I think the azure-pipelines CI is much better than Travis; I see no reason to continue using the service going forward.

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Closes #1658 

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/LTS.md [hunk 1]
```diff
@@ -42,7 +42,6 @@ A "month" is to be a period of 30 consecutive days.
 
 | CI              | OS      | Version        | Package Manager | Node.js        |
 | :-------------- | :------ | :------------- | :-------------- | :------------- |
-| Travis          | Linux   | Ubuntu 14.04   | npm             | 6,8,9,10       |
 | Azure pipelines | Linux   | Ubuntu 16.04   | yarn            | ~~6¹~~,8,10,11 |
 | Azure pipelines | Windows | vs2017-win2016 | npm             | 6,8,10,11      |
```
Comment (mcollina) [inline docs/LTS.md]: I think we are testing on Mac as well now. Also, more node versions.

**Comment by mcollina** [comment]
Can you also do a similar PR for v1?

**Comment by mcollina** [comment]
I would like @jsumners to have full access before we land this: https://github.com/fastify/fastify/issues/1658#issuecomment-494987572.

**Comment by jsumners** [comment]
Please don't let me hold you up on whatever CI integration. I have personal issues with using the Azure CI product and will likely never again bother trying to login.

**Comment by Ethan-Arrowood** [comment]
#1671 is now solved 👍

**Comment by Ethan-Arrowood** [comment]
Rebase after #1682 lands

**Comment by Ethan-Arrowood** [comment]
After CI this is ready to go along with the backport #1685

File: docs/LTS.md [hunk 2]
```diff
@@ -31,19 +31,19 @@ A "month" is to be a period of 30 consecutive days.
...
-| CI              | OS      | Version        | Package Manager | Node.js        |
-| :-------------- | :------ | :------------- | :-------------- | :------------- |
-| Travis          | Linux   | Ubuntu 14.04   | npm             | 6,8,9,10       |
-| Azure pipelines | Linux   | Ubuntu 16.04   | yarn            | ~~6¹~~,8,10,11 |
-| Azure pipelines | Windows | vs2017-win2016 | npm             | 6,8,10,11      |
+| CI              | OS      | Version        | Package Manager | Node.js               |
+| :-------------- | :------ | :------------- | :-------------- | :-------------------- |
+| Azure pipelines | Linux   | Ubuntu 16.04   | npm, yarn       | ~~6¹~~, 8, 10, 11, 12 |
+| Azure pipelines | Windows | vs2017-win2016 | npm, yarn       | 6, 8, 10, 11, 12      |
+| Azure pipelines | Mac     | macOS 10.14    | npm, yarn       | 6, 8, 10, 11, 12      |
```
Comment (mcollina) [inline docs/LTS.md]: the same warning about yarn must be included in Windows and Mac

**Pull request merged**